lib/features/reps_videos/reps_videos.features.inc

```diff
@@ -27,6 +27,39 @@ function reps_videos_views_api($module = NULL, $api = NULL) {
   return array("api" => "3.0");
 }
 
+/**
+ * Implements hook_fe_nodequeue_export_fields().
+ */
+function reps_videos_fe_nodequeue_export_fields() {
+  $nodequeues = array();
+
+  // Exported nodequeues: videos
+  $nodequeues['videos'] = (object) array(
```

Comment:
Remove the (object) this is generating error.
It's something I've noted when I implemented the 3 first queues